cli/README.md

```diff
@@ -69,3 +70,51 @@ Global Flags:
 -kubeconfig, -c=<string>
 	Path to kubeconfig file
 ```
+
+### consul-k8s uninstall
+This command uninstalls Consul on Kubernetes, while prompting whether to uninstall the release and whether to delete all
+related resources such as PVCs, Secrets, and ServiceAccounts.
+
+Get started with:
+```bash
+consul-k8s uninstall
+```
+
+```
+Usage: kubectl consul uninstall [options]
+
+Uninstall Consul and clean up all data.
+Any data store in Consul will not be recoverable.
```

Comment:
```suggestion
Any data stored in Consul will not be recoverable.
```

Reply:
Ended up removing this sentence because of the options we discussed in eng sync!